### ISSUE TITLE
[Question] useMain clarification

### ISSUE THREAD

**Issue by nichevulcan20** [GiorgioBrux/nitro-sniper-enhanced#109]
hey I did this

- `useMain = true`
soo can I leave shit servers from my main account? I mean I will be staying in only 4 - 5 servers from the main account I don't wanna mess my account. and I will join 100 servers with my alt accounts. will it be fine? will my main still get nitro?

**Comment by nichevulcan20** [comment]
bruh am sorry that was a question and it got the bug label

**Comment by nichevulcan20** [comment]
hey does this work?

**Comment by nichevulcan20** [comment]
i mean the situation which is entioned
in  message

**Comment by GiorgioBrux** [comment]
Your main account will always get nitro. Setting useMain to true only means that your account is also used for sniping. If you are only on 4-5 servers and have a bunch of alt accounts it's probably not worth sniping from main.

**Comment by nichevulcan20** [comment]
> Your main account will always get nitro. Setting useMain to true only means that your main account is also used for sniping. If you are only on 4-5 servers and have a bunch of alt accounts it's probably not worth sniping from main.

the extract, i mean

my main wont join any servers for sniping
BUt my alts will join servers so will my alts redeem to main account when go nitro?

**Comment by GiorgioBrux** [comment]
There is no automatic server joiner.
If useMain is set to true the main account is also used for sniping aka if someone sends a gift in the 4-5 servers you are in it will get sniped. Simple as that. All gifts are always redeemed on main.

**Comment by nichevulcan20** [comment]
> There is no automatic server joiner.
> If useMain is set to true the main account is also used for sniping aka if someone sends a gift in the 4-5 servers you are in it will get sniped. Simple as that. All gifts are always redeemed on main.

bro
try to understand what am saying pls

Main account
- have joined servers of just friends
- no clutter servers for siping

Alt account

- have joined 100 large servers
- joined for snipng

Settings

- useMain is set to true
- bot the tokens (alts and main) are entered

Question-
will the main account get nitro from ALTS? because useMain is set to true. 
Will alts actually redeem to main account in this situation bro?

**Comment by GiorgioBrux** [comment]
I've answered your question multiple times. **All gifts are always redeemed on main.**
What useMain does is use your main account for sniping too. If you have alts and are only in a very few servers on your main just set useMain to false.

**Comment by ItzBoiz** [comment]
hmmm idk if he understood this but i got to know about this xd

> I've answered your question multiple times. **All gifts are always redeemed on main.**
> What useMain does is use your main account for sniping too. If you have alts and are only in a very few servers on your main just set useMain to false.

**Comment by GiorgioBrux** [comment]
Alright, I will try to make it as simple as possible.
Using an account for sniping means that that the program logs in the account and looks for codes in the messages that the account receives. All codes are then redeemed on your main. What useMain does is use your account for sniping too. I've made it optional because there is an increased risk that your account gets termed (albeit it's still very low) and so I've leaved it as a choice.
